<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="windowModality">
      <enum>Qt::NonModal</enum>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1067</width>
        <height>903</height>
      </rect>
    </property>
    <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
    </property>
    <property name="windowTitle">
      <string>Remote Streaming App SSH</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
              <widget class="QTabWidget" name="operatorTabs">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="currentIndex">
                  <number>0</number>
                </property>
                <property name="documentMode">
                  <bool>false</bool>
                </property>
                <property name="tabsClosable">
                  <bool>false</bool>
                </property>
                <property name="movable">
                  <bool>false</bool>
                </property>
                <widget class="QWidget" name="tabConfig">
                  <attribute name="title">
                    <string>Configuration</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                      <widget class="QTreeWidget" name="treeConfig">
                        <property name="autoFillBackground">
                          <bool>false</bool>
                        </property>
                        <property name="columnCount">
                          <number>2</number>
                        </property>
                        <column>
                          <property name="text">
                            <string notr="true">1</string>
                          </property>
                        </column>
                        <column>
                          <property name="text">
                            <string notr="true">2</string>
                          </property>
                        </column>
                      </widget>
                    </item>
                    <item>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                          <spacer name="horizontalSpacer">
                            <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                              <size>
                                <width>40</width>
                                <height>20</height>
                              </size>
                            </property>
                          </spacer>
                        </item>
                        <item>
                          <widget class="QPushButton" name="saveButton">
                            <property name="text">
                              <string>Save</string>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </widget>
                <widget class="QWidget" name="tabStats">
                  <attribute name="title">
                    <string>Statistics</string>
                  </attribute>
                  <layout class="QGridLayout" name="gridLayout_3">
                    <item row="0" column="0">
                      <layout class="QVBoxLayout" name="verticalLayout_8">
                        <item>
                          <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                              <layout class="QVBoxLayout" name="verticalLayout_6">
                                <property name="sizeConstraint">
                                  <enum>QLayout::SetMinimumSize</enum>
                                </property>
                                <item>
                                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <property name="sizeConstraint">
                                      <enum>QLayout::SetFixedSize</enum>
                                    </property>
                                    <item>
                                      <spacer name="horizontalSpacer_6">
                                        <property name="orientation">
                                          <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeType">
                                          <enum>QSizePolicy::Fixed</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                          <size>
                                            <width>3</width>
                                            <height>20</height>
                                          </size>
                                        </property>
                                      </spacer>
                                    </item>
                                    <item>
                                      <widget class="QCheckBox" name="checkAllBox">
                                        <property name="text">
                                          <string/>
                                        </property>
                                      </widget>
                                    </item>
                                    <item>
                                      <spacer name="horizontalSpacer_5">
                                        <property name="orientation">
                                          <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeType">
                                          <enum>QSizePolicy::Fixed</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                          <size>
                                            <width>80</width>
                                            <height>20</height>
                                          </size>
                                        </property>
                                      </spacer>
                                    </item>
                                    <item>
                                      <widget class="QPushButton" name="approveButton">
                                        <property name="font">
                                          <font>
                                            <underline>true</underline>
                                          </font>
                                        </property>
                                        <property name="styleSheet">
                                          <string notr="true">color: rgb(48, 12, 255);</string>
                                        </property>
                                        <property name="text">
                                          <string>Approve</string>
                                        </property>
                                        <property name="flat">
                                          <bool>true</bool>
                                        </property>
                                      </widget>
                                    </item>
                                    <item>
                                      <widget class="QPushButton" name="kickButton">
                                        <property name="font">
                                          <font>
                                            <underline>true</underline>
                                          </font>
                                        </property>
                                        <property name="cursor">
                                          <cursorShape>PointingHandCursor</cursorShape>
                                        </property>
                                        <property name="styleSheet">
                                          <string notr="true">color: rgb(48, 12, 255);</string>
                                        </property>
                                        <property name="text">
                                          <string>Kick</string>
                                        </property>
                                        <property name="flat">
                                          <bool>true</bool>
                                        </property>
                                      </widget>
                                    </item>
                                    <item>
                                      <spacer name="horizontalSpacer_2">
                                        <property name="orientation">
                                          <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property name="sizeHint" stdset="0">
                                          <size>
                                            <width>40</width>
                                            <height>20</height>
                                          </size>
                                        </property>
                                      </spacer>
                                    </item>
                                  </layout>
                                </item>
                                <item>
                                  <widget class="QListWidget" name="usersList">
                                    <property name="sizePolicy">
                                      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                      </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                      <string notr="true">
                                        QListView::item {
                                        border: 0px;
                                        padding-left: 2px;
                                        }
                                      </string>
                                    </property>
                                    <property name="editTriggers">
                                      <set>QAbstractItemView::NoEditTriggers</set>
                                    </property>
                                    <property name="selectionMode">
                                      <enum>QAbstractItemView::NoSelection</enum>
                                    </property>
                                    <property name="viewMode">
                                      <enum>QListView::ListMode</enum>
                                    </property>
                                    <property name="selectionRectVisible">
                                      <bool>false</bool>
                                    </property>
                                  </widget>
                                </item>
                              </layout>
                            </item>
                            <item>
                              <spacer name="horizontalSpacer_4">
                                <property name="orientation">
                                  <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                  <size>
                                    <width>40</width>
                                    <height>20</height>
                                  </size>
                                </property>
                              </spacer>
                            </item>
                          </layout>
                        </item>
                        <item>
                          <layout class="QVBoxLayout" name="verticalLayout_7">
                            <item>
                              <layout class="QGridLayout" name="gridLayout">
                                <item row="0" column="0">
                                  <widget class="QLabel" name="label_2">
                                    <property name="text">
                                      <string>Bandwidth usage</string>
                                    </property>
                                  </widget>
                                </item>
                                <item row="0" column="1">
                                  <spacer name="horizontalSpacer_3">
                                    <property name="orientation">
                                      <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                      <size>
                                        <width>40</width>
                                        <height>20</height>
                                      </size>
                                    </property>
                                  </spacer>
                                </item>
                              </layout>
                            </item>
                            <item>
                              <widget class="QGraphicsView" name="graphicsView"/>
                            </item>
                          </layout>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </widget>
                <widget class="QWidget" name="tabVideo">
                  <attribute name="title">
                    <string>Video</string>
                  </attribute>
                </widget>
              </widget>
            </item>
            <item>
              <widget class="QTabWidget" name="usersTab">
                <property name="currentIndex">
                  <number>0</number>
                </property>
                <widget class="QWidget" name="videoTab">
                  <attribute name="title">
                    <string>Video</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                      <widget class="QLabel" name="labelNoConnection">
                        <property name="styleSheet">
                          <string notr="true">color: rgb(195, 192, 198);</string>
                        </property>
                        <property name="text">
                          <string>Connection is not established</string>
                        </property>
                        <property name="alignment">
                          <set>Qt::AlignCenter</set>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </widget>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
    <widget class="QDockWidget" name="dockWidget">
      <property name="minimumSize">
        <size>
          <width>180</width>
          <height>213</height>
        </size>
      </property>
      <property name="features">
        <set>QDockWidget::DockWidgetFloatable</set>
      </property>
      <property name="allowedAreas">
        <set>Qt::LeftDockWidgetArea</set>
      </property>
      <property name="windowTitle">
        <string>Connection</string>
      </property>
      <attribute name="dockWidgetArea">
        <number>1</number>
      </attribute>
      <widget class="QWidget" name="dockWidgetContents">
        <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
            <layout class="QVBoxLayout" name="verticalLayout">
              <item>
                <widget class="QLabel" name="label">
                  <property name="text">
                    <string>Remote Servers:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QListWidget" name="serversList"/>
              </item>
              <item>
                <widget class="QPushButton" name="connectButton">
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <property name="text">
                    <string/>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="operatorCheckBox">
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <property name="text">
                    <string>as Operator</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </widget>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources/>
  <connections/>
</ui>
